src/components/search-box.vue

```diff
@@ -16,9 +16,9 @@
 
     <SearchButton
       v-if="isDesktopOrGreater || !$x.query.searchBox"
-      class="x-input-group-button-primary"
+      class="x-input-group-button-primary x-bg-neutral-90"
```

Comment:
This doesn't work design wise, we don't have a button with a background color of `neutral-90`. We lose the hovering interactions with this.